utils/src/main/kotlin/ProcessCapture.kt

```diff
@@ -43,6 +43,8 @@ class ProcessCapture(workingDir: File?, environment: Map<String, String>, vararg
     val stdoutFile = File(tempDir, "$tempPrefix.stdout")
     val stderrFile = File(tempDir, "$tempPrefix.stderr")
 
+    private val DEBUG_LINES = 20
```

Comment:
Why not make this an optional constructor parameter?

Reply:
I believe this is too much of an internal implementation detail to be exposed at that level. For now at least 😉